web/mta_highlighting/generate-lua-tmlanguage.js

```diff
@@ -11,6 +11,8 @@ const functionsDir = path.resolve(__dirname, '../../functions');
 const basePath = path.resolve(__dirname, './lua-base.tmLanguage.json');
 const outputPath = path.resolve(__dirname, '../src/grammars/lua-mta.tmLanguage.json');
 
+const mtaKeywords = ['string','bool','boolean','number','int','float','element','player','vehicle','ped','object','building'];
```

Comment:
Good job! But the element related key words should be fetched from the elements folder. Each element has a name attribute that will correspond to a page "/elementName" directly.

Reply:
`'string','bool','boolean','number','int','float','element'`: those can be hardcoded 
The rest => from yaml

Reply:
I think a separate directory, like "types," with individual pages would be better than that static array

Reply:
> I think a separate directory, like "types," with individual pages would be better than that static array

? There is an elements directory containing all the element/class types in yaml

Reply:
> > I think a separate directory, like "types," with individual pages would be better than that static array
> 
> ? There is an elements directory containing all the element/class types in yaml

I mean types like string, bool, int etc

Reply:
> > > I think a separate directory, like "types," with individual pages would be better than that static array
> > 
> > 
> > ? There is an elements directory containing all the element/class types in yaml
> 
> I mean types like string, bool, int etc

Oh I agree we can do that later